File: Challenge 3/Biruk-3c.py
# Write a function in Python that accepts a string and returns the number of vowels in the string.

strn = input("Enter text: ").lower()

def vowels(text):
    x = 0
    for i in text:
        if i in ['a','e','i','o','u']:
            x+=1
    return x

print(vowels(strn))

File: JavaScript/Challenge 3/Biruk-3c.js
// Write a function in Python that accepts a string and returns the number of vowels in the string.

console.log("Challenge 3c")
console.log()

let strn = prompt("Enter text").toLowerCase()

function vowels(text){
    let x = 0
    for (let i=0; i<text.length; i++) {
        if(['a','e','i','o','u'].includes(text[i])){
            x+=1
        }
    }
    return x    
}

console.log(vowels(strn))